cfgov/apache/conf.d/alias.conf

```diff
@@ -0,0 +1,13 @@
+# Bing site auth
+Alias /BingSiteAuth.xml ${STATIC_PATH}/BingSiteAuth.xml
+Alias /code.json ${STATIC_PATH}/code.json
+Alias /data.json ${STATIC_PATH}/data.json
+Alias /favicon.ico ${STATIC_PATH}/favicon.ico
+Alias /foia/quarterly ${STATIC_PATH}/foia/quarterly
+# Facebook site auth
+Alias /lln8595c61g9qnvuwvtlcwo1k6kem8.html ${STATIC_PATH}/lln8595c61g9qnvuwvtlcwo1k6kem8.html
+Alias /robots.txt ${STATIC_PATH}/robots.txt
+Alias /static/f/ ${STATIC_PATH}/nemo/_/f/
```

Comment:
We need to add entries for the Akamai SureRoute test objects, as implemented over in ansible#316.

Reply:
Added